opennms-webapp/src/main/java/org/opennms/web/controller/alarm/AlarmDetailController.java

```diff
@@ -112,23 +129,26 @@ public ModelAndView detail(HttpServletRequest httpServletRequest, HttpServletRes
             acknowledgments = m_webAlarmRepository.getAcknowledgments(alarmId);
 
             // Get alarm by ID
-            m_alarm = m_webAlarmRepository.getAlarm(alarmId);
-            logger.debug("Alarm retrieved: '{}'", m_alarm.toString());
+            alarm = m_webAlarmRepository.getAlarm(alarmId);
+            logger.debug("Alarm retrieved: '{}'", alarm.toString());
         } catch (NumberFormatException e) {
             logger.error("Could not parse alarm ID '{}' to integer.", safeRequest.getParameter("id"));
         } catch (Throwable e) {
             logger.error("Could not retrieve alarm from webAlarmRepository for ID='{}'", alarmIdString);
         }
 
-        if (m_alarm == null) {
+        if (alarm == null) {
             throw new AlarmIdNotFoundException("Could not find alarm with ID: " + alarmIdString, alarmIdString);
         }
 
         // return to view WEB-INF/jsp/alarm/detail.jsp
         ModelAndView mv = new ModelAndView("alarm/detail");
-        mv.addObject("alarm", m_alarm);
+        mv.addObject("alarm", alarm);
         mv.addObject("alarmId", alarmIdString);
         mv.addObject("acknowledgments", acknowledgments);
+        if (Boolean.getBoolean("org.opennms.web.console.alarms.relatedEvents")) {
```

Comment:
Can you add an entry in `opennms-base-assembly/src/main/filtered/etc/opennms.properties` with a short description and the default value?

Reply:
Oops, this branch was supposed to be based off `foundation-2019`, which is why I didn't change `opennms.properties` by default.  I was gonna do that when it merged forward to develop.